pkg/assembler/graphql/examples/certify_good.gql

```diff
@@ -0,0 +1,130 @@
+fragment allArtifactTree on Artifact {
```

Comment:
should reuse artifact tree fragment? (this will unfortunately cause my gql codegen for UI stuff to fail)